tools/umi_tools/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM continuumio/miniconda3:4.8.2
+LABEL authors="chris.cheshire@crick.ac.uk" \
+      description="Docker image containing all requirements for the nf-core umi_tools module"
+
+# Install apt packages
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+ procps=2:3.3.15-2 \
```

Comment:
Is this a required dependency for `umitools`?

Reply:
what `procps`?

Reply:
No, this is for Nextflow. We already have this in the base `nf-core` image, so I would prefer to use that instead: https://github.com/nf-core/tools/blob/master/Dockerfile

Reply:
no worries

Reply:
I removed this statement, there is just the conda install now - PR raised for change to the base image